src/mvba/broadcaster.rs

```diff
@@ -2,55 +2,60 @@ use super::{bundle::Bundle, NodeId};
 use blsttc::{PublicKeyShare, SecretKeyShare};
 
 // Broadcaster holds information required to broadcast the messages.
-// If a node is an observer node, it doesn't broadcast messages.
-// For observer node SecretKeyShare and Node ID set to None
+// If a node is an observer node, it should't broadcast messages.
+// TODO: How to find out it is an observer node or not?
 pub struct Broadcaster {
     bundle_id: u32,
-    secret_key: SecretKeyShare,
-    node_id: Option<NodeId>,
-    bundles: Vec<Bundle>,
+    _sec_key_share: SecretKeyShare,
```

Comment:
Any reason for the underscore in front here?

Reply:
To stop clippy from nagging to much!

Reply:
Is it still complaining? You might not have been using that field at the time, but I see you ware using it now.

Reply:
I add it in TODO list. If we want to sign the whole bundle (it looks it is not necessary) we need it. Later we decide about it.